doc/Comments.xml

```diff
@@ -343,6 +343,10 @@ use @ExampleSession:
 ]]></Listing>
 It inserts an example into the manual just as @Example would do, but all lines are commented
 and therefore not executed when the file is read.
+All lines have to start with an &AutoDoc; comment
```

Comment:
Super nitpick, but why the weird line wrapping, and why end the line here? You move the last "word" of the sentence to a new line, when you then have a muuuch longer line coming next...